mlrun/track/trackers/mlflow_tracker.py

```diff
@@ -0,0 +1,449 @@
+                ):
+                    model_paths.append(artifact_local_path)
+                else:
+                    # Log the artifact:
+                    artifact = MLFlowTracker._log_artifact(
+                        context_or_project=context,
+                        key=pathlib.Path(artifact.path).name.replace(".", "_"),
+                        local_path=artifact_local_path,
+                        tmp_path=tmp_dir,
+                    )
+                    artifacts[artifact.key] = artifact
+            if model_paths:
+                for model_path in model_paths:
+                    MLFlowTracker._log_model(
+                        context_or_project=context,
+                        model_uri=model_path,
+                        key=pathlib.Path(model_path).stem,
+                        metrics=results,
+                        extra_data=artifacts,
+                        tmp_path=tmp_dir,
+                    )
+
+    @staticmethod
+    def _log_model(
+        context_or_project: Union[MLClientCtx, MlrunProject],
+        model_uri: str,
+        key: str,
+        metrics: dict,
+        extra_data: dict,
+        tmp_path: os.path,
+    ):
+        """
+        Log the given produced model from MLFlow as a model artifact in MLRun.
+
+        :param context_or_project: The MLRun context or project to log to.
+        :param model_uri:          The local path to the model (an MLFlow model directory locally downloaded).
+        :param key:                The model artifact's key.
+        :param metrics:            The key/value dict of model metrics
+        :param extra_data:         The extra data to log in addition to the model (training data for example)
+        :param tmp_path:           The path to the dir where we temporarily save model and artifacts
+        """
+        # Get the model info from MLFlow:
+        model_info = mlflow.models.get_model_info(model_uri=model_uri)
+
+        # Prepare the archive path:
+        model_uri = pathlib.Path(model_uri)
+        archive_path = pathlib.Path(tmp_path) / f"{model_uri.stem}.zip"
+
+        # Zip the artifact:
+        with zipfile.ZipFile(archive_path, "w") as zip_file:
+            for path in model_uri.rglob("*"):
+                zip_file.write(filename=path, arcname=path.relative_to(model_uri))
+
+        # Get inputs and outputs info:
+        inputs = outputs = None
+        if model_info.signature is not None:
+            if model_info.signature.inputs is not None:
+                inputs = MLFlowTracker._schema_to_feature(
+                    schema=model_info.signature.inputs
+                )
+            if model_info.signature.outputs is not None:
+                outputs = MLFlowTracker._schema_to_feature(
+                    schema=model_info.signature.outputs
+                )
+
+        # Log the model:
+        return context_or_project.log_model(
+            key=key,
+            framework="mlflow",
```

Comment:
The framework is usually the actual model framework (for example `sklearn`). From my understanding of `mlflow`, it has a format for logging models, but also it knows what framework actually generated the model itself. For example, a model can be of type `mlflow.sklearn`. Do we have any way to know this and mark it here? I know the `framework` parameter is usually not triggering any functionality, but it might be nice to give this data to the user without having to dig into the model itself.

Reply:
i can access it but it isn't always available (also not very accessible), how do suggest i call it? in mlflow it's referred to as: "loader_module"

Reply:
added a list of flavors to model tags for users to have knowlege about the models frameworks (multiple frameworks aew possible in mlflow)